src/hooks/pager.ts

```diff
@@ -8,39 +8,60 @@ import { useService } from './service';
 
 export interface PagerManager {
     loadNext: () => void;
+    loadPrevious: () => void;
+    loadPage: (page: number) => void;
     reload: () => void;
     hasNext: boolean;
+    hasPrevious: boolean;
+    currentPage: number;
 }
 
 export function usePager<T extends AidboxResource>(
     resourceType: T['resourceType'],
     resourcesOnPage: number = 15,
     searchParams: SearchParams = {}
 ): [RemoteData<Bundle<T>>, PagerManager] {
-    const [pageToLoad, setPageToLoad] = useState(1);
+    const [pageToLoad, setPageToLoad] = useState(searchParams._page ? Number(searchParams._page) : 1);
     const [reloadsCount, setReloadsCount] = useState(0);
-
     const [resources] = useService(
         () =>
             getFHIRResources(resourceType, {
                 ...searchParams,
                 _count: resourcesOnPage,
                 _page: pageToLoad,
             }),
-        [pageToLoad, reloadsCount]
+        [pageToLoad, reloadsCount, resourcesOnPage]
     );
 
+    const loadNext = useCallback(() => setPageToLoad((currentPage) => currentPage + 1), []);
+
+    const loadPrevious = useCallback(() => setPageToLoad((currentPage) => currentPage - 1), []);
+
+    const loadPage = useCallback((page: number) => setPageToLoad(page), []);
```

Comment:
We can probably remove this callback as it only calls internal function with all of its arguments forwarded as they are (without any transformation being applied). I believe we can use `setPageToLoad` directly (setting `loadPage: setPageToLoad` in the return object).